packages/terra-date-input/src/DateInput.jsx

```diff
@@ -654,20 +661,33 @@ class DateInput extends React.Component {
       dateValue = `${year}-${month}-${day}`;
     }
 
+    let format = `(${this.props.intl.formatMessage({ id: 'Terra.date.input.dayPlaceholder' })} ${this.props.intl.formatMessage({ id: 'Terra.date.input.monthLabel' })} ${this.props.intl.formatMessage({ id: 'Terra.date.input.yearPlaceholder' })})`;
```

Comment:
nit: This can be added in an `else` below so it doesn't always get computed and overridden unnecessarily if the format is `month-date-year`.

Reply:
Updated here https://github.com/cerner/terra-framework/pull/1230/commits/a6bc9e0c450a8c6e40868176fa10ef8beb851c09